lib/utils/preferences.dart

```diff
@@ -24,6 +25,64 @@ Future<void> performSharedPreferencesMigration() async {
   bool? legacyCommentUseColorizedUsername = prefs.getBool(LocalSettings.commentUseColorizedUsername.name);
   if (legacyCommentUseColorizedUsername != null) {
     await prefs.remove(LocalSettings.commentUseColorizedUsername.name);
-    await prefs.setBool(LocalSettings.userFullNameColorizeUserName.name, legacyCommentUseColorizedUsername);
+    await prefs.setBool('user_fullname_colorize_user_name', legacyCommentUseColorizedUsername);

```

Comment:
We can replace this with `await prefs.setString(LocalSettings.userFullNameUserNameColor.name, const NameColor.fromString(color: NameColor.themePrimary).color);` if we're not worried about doing the migrations below

Reply:
Agreed! I think we can set it only if the old value was `true`. I also thing we can use the string value directly without having to construct the `NameColor`. Let me know what you think of the latest.